src/components/menus/ViewMenu.svelte

```diff
@@ -2,50 +2,75 @@
-  import SaveIcon from 'bootstrap-icons/icons/save.svg?component';
+  import ViewGridIcon from '@nasa-jpl/stellar/icons/view_grid_filled.svg?component';
+  import { createEventDispatcher } from 'svelte';
   import { user as userStore } from '../../stores/app';
-  import { view } from '../../stores/views';
-  import effects from '../../utilities/effects';
+  import { view, viewIsModified } from '../../stores/views';
+  import { showSavedViewsModal } from '../../utilities/modal';
+  import { Status } from '../../utilities/status';
+  import NavButton from '../app/NavButton.svelte';
   import Menu from './Menu.svelte';
   import MenuItem from './MenuItem.svelte';
 
+  const defaultViewName = 'Default View';
+  const dispatch = createEventDispatcher();
+
   let saveViewDisabled: boolean = true;
   let viewMenu: Menu;
 
-  $: saveViewDisabled = $view?.name === '' || $view?.owner !== $userStore?.id;
+  $: saveViewDisabled = $view?.name === '' || $view?.owner !== $userStore?.id || !$viewIsModified;
+
+  function saveAsView() {
+    if ($view) {
+      dispatch('create-view', { definition: $view.definition, owner: $userStore?.id });
+    }
+  }
 
-  function createView() {
+  function editView() {
     if ($view) {
-      effects.createView($userStore?.id, $view.definition);
+      dispatch('edit-view', { definition: $view.definition, owner: $userStore?.id });
     }
   }
 
   function saveView() {
     if ($view && $view.owner === $userStore?.id && !saveViewDisabled) {
-      effects.updateView($view.id, { definition: $view.definition, name: $view.name });
+      dispatch('save-view', { definition: $view.definition, id: $view.id, name: $view.name });
+    }
+  }
+
+  function resetView() {
+    if ($view) {
+      dispatch('reset-view');
     }
   }
 </script>
 
 <!-- svelte-ignore a11y-click-events-have-key-events -->
-<div class="view-menu st-typography-body" on:click|stopPropagation={() => viewMenu.toggle()}>
-  <ChevronDownIcon />
-
-  <Menu bind:this={viewMenu} offset={[-93, -5]}>
-    <MenuItem on:click={createView}>
-      <PlusIcon />
-      Create View
-    </MenuItem>
-    <MenuItem disabled={saveViewDisabled} on:click={saveView}>
-      <SaveIcon />
-      Save View
-    </MenuItem>
-  </Menu>
-</div>
+<NavButton
+  status={$viewIsModified ? Status.Modified : null}
+  title={$view?.name ?? defaultViewName}
+  on:click={() => viewMenu.toggle()}
+>
+  <ViewGridIcon />
+  <div class="view-menu st-typography-body" slot="menu">
+    <ChevronDownIcon />
+
+    <Menu bind:this={viewMenu} offset={[-120, -5]}>
+      <MenuItem disabled={saveViewDisabled} on:click={saveView}>Save</MenuItem>
+      <MenuItem on:click={saveAsView}>Save as</MenuItem>
+      <MenuItem disabled={!$viewIsModified} on:click={resetView}>Reset to default</MenuItem>
+      <MenuItem on:click={showSavedViewsModal}>Browse saved views</MenuItem>
+      {#if $view?.name && $view.name !== defaultViewName}
+        <hr />
```

Comment:
Could we lighten the color on the <hr>? 
<img width="178" alt="image" src="https://user-images.githubusercontent.com/4419607/216725484-0f0a4b1b-089a-4b7b-9e27-18abff08b406.png">